code/modules/cargo/packs/_packs.dm

```diff
@@ -122,3 +122,15 @@
 	name = "[purchaser]'s Materials Order"
 	src.cost = cost
 	src.contains = contains
+
+/datum/supply_pack/custom/minerals/fill(obj/structure/closet/crate/C)
+	. = ..()
+	//Remove our material sheets from SSstock_market's materials_quantity equal to the quantity within the crate.
+	for(var/obj/item/stack/sheet/possible_stack as anything in contains)
+		if(!ispath(possible_stack, /obj/item/stack/sheet))
+			continue
```

Comment:
```suggestion
	for(var/obj/item/stack/sheet/possible_stack as anything in contains) 
```

No need `is_path()` check since this order will always contain sheets only. Safe assumption since this is used only in the  market console and no where else

Reply:
Actually, dismiss this, I'm goanna help you out. One sec